uzbl/ext.py

```diff
@@ -38,12 +38,16 @@ def __getitem__(self, owner):
 class BasePlugin(object):
     """Base class for all uzbl plugins"""
 
+    def config_section(self):
+        return 'plugin.%s' % getattr(self, 'CONFIG_SECTION', self.__class__.__name__)
```

Comment:
I think this might be some unnecessary magic. I think we can require the plugins to a have fully qualified section explicitly defined.